node/parallel-dev/src/chain_spec.rs

```diff
@@ -210,29 +211,10 @@ fn testnet_genesis(
             currencies: vec![CurrencyId::KSM, CurrencyId::USDT, CurrencyId::xKSM],
             borrow_index: Rate::one(),                             // 1
             exchange_rate: Rate::saturating_from_rational(2, 100), // 0.02
-            base_rate: Rate::saturating_from_rational(2, 100),     // 2%
-            jump_rate: Rate::saturating_from_rational(10, 100),    // 10%
-            full_rate: Rate::saturating_from_rational(32, 100),    // 32%
-            jump_utilization: Ratio::from_percent(80),             // 80%
-            collateral_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(50)),
-                (CurrencyId::USDT, Ratio::from_percent(50)),
-                (CurrencyId::xKSM, Ratio::from_percent(50)),
-            ],
-            liquidation_incentive: vec![
-                (CurrencyId::KSM, Rate::saturating_from_rational(110, 100)),
-                (CurrencyId::USDT, Rate::saturating_from_rational(110, 100)),
-                (CurrencyId::xKSM, Rate::saturating_from_rational(110, 100)),
-            ],
-            close_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(50)),
-                (CurrencyId::USDT, Ratio::from_percent(50)),
-                (CurrencyId::xKSM, Ratio::from_percent(50)),
-            ],
-            reserve_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(15)),
-                (CurrencyId::USDT, Ratio::from_percent(15)),
-                (CurrencyId::xKSM, Ratio::from_percent(15)),
+            markets: vec![
+                (CurrencyId::KSM, MARKET_MOCK),
```

Comment:
use the literal struct instead of MARKET_MOCK for we will set different parameters for each market.

Reply:
Replaced